Tests/SwiftFormatTests/PrettyPrint/ClosureExprTests.swift

```diff
@@ -251,12 +251,11 @@ final class ClosureExprTests: PrettyPrintTestCase {
       let a = funcCall() { [weak self] (a: Int) in
         return a + 1
       }
-      let a = funcCall() {
-        [weak self, weak a = self.b] (a: Int) in
+      let a = funcCall() { [weak self, weak a=self.b] (a: Int) in
```

Comment:
I think you need to rebase your local checkout or make sure that you have the latest dependencies; these shouldn't change based on https://github.com/swiftlang/swift-format/pull/790 and the corresponding swift-syntax PR.